torch/csrc/jit/tensorexpr/kernel.cpp

```diff
@@ -3296,6 +3318,7 @@ void TensorExprKernel::compile() {
 
 TensorExprKernel::TensorExprKernel(const std::shared_ptr<Graph>& subgraph)
     : graph_(subgraph), code_(subgraph, "") {
+  sanitizeName(subgraph->block());
```

Comment:
It would be better to call this from `compile()` just to keep all the transformations in one place.